src/main/resources/assets/bashfiremakerock/lang/de_de.lang

```diff
@@ -0,0 +1,9 @@
+config.bashfiremakerock:title=Hauptmenü
+config.bashfiremakerock:chance=Chance auf Feuer
+config.bashfiremakerock:blocks=Blöcke
+config.bashfiremakerock:blocks.allow=Erlaubte Liste
+config.bashfiremakerock:start=Minimum an Klicks für Feuer
+config.bashfiremakerock:delay.min=Verzögerung zwischen Klicks
+config.bashfiremakerock:delay.max=Verzögerung um Klicks zu merken
+
+bashfiremakerock:error.event.click=Fehlerhandlungsereignis!! Benachrichtige einen Administrator, den Konosolenlog einzusehen.
```

Comment:
The `Handling` in `Error Handling Click Event` is a verb. Should be something like `Fehler beim Verarbeiten des Klicks`

Reply:
Done